docs/admin/setup/pyris.rst

```diff
@@ -9,17 +9,11 @@ Prerequisites
 ^^^^^^^^^^^^^
 
 - A server/VM or local machine
-- Docker installed on the machine
+- Docker installed on the machine (Required for Docker Setup)
 
 Installation Steps
 ^^^^^^^^^^^^^^^^^^
 
-Create a Directory for Pyris Deployment
-"""""""""""""""""""""""""""""""""""""""
-
-Create a directory where you will deploy Pyris. For example, you can create a directory at ``/opt/pyris`` on the machine.
-For local development, you can create a directory at ``~/pyris``.
-
 Clone the Pyris Repository
```

Comment:
That's only needed for development. For Production you simply add a docker compose entry

Reply:
I think, we need to clone the Pyris repository regardless if we use it in a development or production environment. However, if you have another workflow, I would gladly adjust the documentation that also covers your workflow 🙂.